apps/web/src/features/layout/components/ThemeEnvironment/phoenixEnvironment.module.css

```
@@ -0,0 +1,488 @@
+}
+
+.wrapper[data-mode="light"] .tooltip {
+  background: rgba(254, 248, 242, 0.94);
+  color: #1c130d;
+  border: 1px solid rgba(234, 88, 12, 0.35);
+  box-shadow: 0 10px 28px rgba(120, 86, 69, 0.18), 0 0 16px rgba(249, 115, 22, 0.12);
+}
+
+.tooltip strong {
+  display: block;
+  font-family: 'Space Grotesk', 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)